config/locales/phrase-exports/pt-BR.yml

```diff
@@ -979,7 +979,3 @@ pt-BR:
         allow_blank: deveria ter o formato de um endereço de e-mail. Por favor utilize
           um endereço diferente ou deixe em branco.
         no_blank: deveria ter o formato de um endereço de e-mail.
-      veracity:
```

Comment:
You should leave out changes to phrase-exports/ files - we let Phrase / the Translation team resolve the differences, for now.

Reply:
Thanks - removed.